nixos/modules/services/misc/gitea.nix

```diff
@@ -483,7 +483,7 @@ in
       description = "gitea";
       after = [ "network.target" ] ++ lib.optional usePostgresql "postgresql.service" ++ lib.optional useMysql "mysql.service";
       wantedBy = [ "multi-user.target" ];
-      path = [ gitea pkgs.git ];
+      path = with pkgs; [ gitea git gnupg ];
```

Comment:
The `with ...` change is misleading. I know that `with` doesn't take precedence over higher `let ... in`, but anyone (including me) who'll read that in a few months will assume that `gitea` comes from `pkgs` until they find the `gitea =` expression at the top and remember scoping rules in Nix.

iow, please don't do that.

Reply:
Thank you for the feedback, will fix.
I tested this change and sadly `gnupg` core dumps with signal 31 `SIGSYS` when `gitea` tries to use it (e.g. creating a merge commit). Do you have an idea what we might be missing?

Reply:
Wild guess, perhaps you need the agent up and running? But I don't really have an idea, anything interesting in the logs? Perhaps I'll get to take a look soonish.

Reply:
Just this:
```
gitea[582840]:         fatal: failed to write commit object
gitea[582840]:         error: gpg failed to sign the data
gitea[582840]:         Error: exit status 128 - error: gpg died of signal 31
gitea[582840]: 2022/11/27 15:40:01 ...s/repository/init.go:347:initRepoCommit() [E] Failed to commit: [commit --author=...
systemd-coredump[589449]: [🡕] Process 589447 (gpg) of user 997 dumped core.
```
I'll try making sure the agent is available for the `gitea` user.

Reply:
Fixed in 6033e4e5ed285759726050faaf6f75fbdbb33b20